examples_tests/41.VisibilityBuffer/main.cpp

```diff
@@ -1229,6 +1250,10 @@ int main()
         driver->clearColorBuffer(EFAP_COLOR_ATTACHMENT0, invalidObjectCode);
         fillVBuffer(sceneData.frustumCulledMdiBuffer);
 
+        // create depth pyramid
+        for (uint32_t i = 0u; i < dpgDsCnt; i++)
+            dpg.generateMipMaps(depthBufferView, dpgPpln, dpgDs[i], dpgPushConstants[i]);
+
         // occlusion cull (against partially filled new Z-buffer)
         driver->setRenderTarget(zBuffOnlyFrameBuffer);
         driver->bindDescriptorSets(video::EPBP_GRAPHICS, cullShaderData.occlusionCullPipeline->getLayout(), 0u, 1u, &cullShaderData.occlusionCullDS.get(), nullptr);
```

Comment:
either generate the depth pyramid or occlusion cull using z-test (also add an option for absolutely no occlusion culling, will be useful to test on chromebook)